addons/website/static/src/less/website.ui.components.less

```diff
@@ -180,67 +208,206 @@ body .modal {
             }
         }
 
-        ~ .modal-backdrop {
+        + .modal-backdrop {
             opacity: 0.8;
         }
     }
 
+    // TOP MENU EDITOR
+    &.o_contentMenu_modal {
+        .oe_menu_editor {
+            ul {
+                padding-left: 37px;
+            }
+
+            li {
+                margin-top: -1px;
+
+                .input-group-addon {
+                    border-radius: 0;
+                }
+            }
+        }
+
+        // 'ADD MENU ITEM' MODAL BACKDROP
+        ~ .modal-backdrop {
+            opacity: 0.3;
+            z-index: @zindex-modal + 1;
+        }
```

Comment:
Mmh, I already made a fix for this: https://github.com/odoo/odoo/blob/master/addons/web/static/src/less/modal.less#L78 but unfortunately, this does not apply to the frontend. I suggest you duplicate my code instead (as it is already the case for lots of modal style because I only want to style the non-visitors modals) until I come up with a better solution.